table/tables/mutation_checker.go

```diff
@@ -36,6 +36,11 @@ type mutation struct {
 	value []byte
 }
 
+type richIndexMutation struct {
+	mutation
+	indexID int64
```

Comment:
Could this filed be saved in the `mutation` so we don't need to build `richIndexMutation` which brings extra cost.